conf/airframes/BR/DelFlyDualPWMservo.xml

```diff
@@ -0,0 +1,276 @@
+<!-- this is a LadyBird quadrotor frame equiped with Lisa/S 1.0 -->
```

Comment:
guess it would make sense to provide proper description here (at least remove the ladybird stuff that is not correct)